api-reference/_hidden/dxTreeListColumnButton/disabled.md

```diff
@@ -5,22 +5,22 @@ default: false
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies whether the button is disabled.
 
 ##### param(options): Object
-<!-- Description goes here -->
+Information about the current row and column that contain the button.
 
 ##### field(options.column): dxTreeListColumn
-<!-- Description goes here -->
+The column's properties.
 
 ##### field(options.component): dxTreeList
-<!-- Description goes here -->
+The UI component's instance.
 
 ##### field(options.row): dxTreeListRowObject
-<!-- Description goes here -->
+The row's properties.
 
 ##### return: Boolean
-<!-- Description goes here -->
+**true** if the button should be disabled; otherwise, **false**.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+Use the function to activate or disable the button for specific rows. See a related example in the **buttons[]**.[visible](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/buttons/#visible) topic.
```

Comment:
Both descriptions are virtually the same. [Use imports for the TreeList](http://tw.devexpress.devx/devextreme-docs-wiki/reuse-content/#imports).